### ISSUE TITLE
how to freely choose the application to open the document on the web or mobile phone.

### ISSUE THREAD

**Issue by bulesky89857** [nextcloud/server#40970]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

I have installed onlyoffcie and nextcloud office at the same time. I have some questions that I would like to ask:
1. The Collabora/code server I built myself can be used normally by directly adding the server address in nextcloud. However, I think this seems unsafe. There is no requirement to enter the key. On the contrary, the onlyoffcie server I built can set the key. .
2.onlyoffcie The open source version of the Android mobile phone version only has the viewing function, while nextcloud office can perform editing functions. This raises a question. How to specify the default document opening application on the mobile phone as nextcloud office?
3. In the nextcloud web version, the right-click option automatically adds the onlyoffice option, but there is no nextcloud office option, and the application for opening the document cannot be specified?
In short, I want to know how to freely choose the application to open the document on the web or mobile phone.

### Steps to reproduce

1.1
2.1
3.11

### Expected behavior

1

### Installation method

Community Web installer on a VPS or web space

### Nextcloud Server version

25

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by joshtrichards** [comment]
> This is a bug, not a question or a configuration/webserver/proxy issue.

Hi @bulesky89857 - These are all good questions, but this is not a technical support channel.

For community-based assistance please use the [Nextcloud Help Forum](https://help.nextcloud.com/). For commercial support [there are options](https://nextcloud.com/enterprise/)
for that as well.

**Comment by bulesky89857** [comment]
> > This is a bug, not a question or a configuration/webserver/proxy issue.
> 
> Hi @bulesky89857 - These are all good questions, but this is not a technical support channel.
> 
> For community-based assistance please use the [Nextcloud Help Forum](https://help.nextcloud.com/). For commercial support [there are options](https://nextcloud.com/enterprise/) for that as well.

ok ,thank you
